xray_vision/mpl_plotting/roi.py

```diff
@@ -42,3 +43,42 @@ def show_label_array(ax, label_array, cmap=None, **kwargs):
                    **kwargs)
 
     return im
+
+
+def show_label_array_on_image(ax, image, label_array, cmap=None,
+                              norm=LogNorm(), **kwargs):
+    """""
+    This will plot the required ROI's(labeled array on the image
+
+    Additional kwargs are passed through to `ax.imshow`.
+    If `vmin` is in kwargs, it is clipped to minimum of 0.5.
+
+    Parameters
+    ----------
+    ax : Axes
+        The `Axes` object to add the artist too
+
+    image : array
+        The image array
+
+    label_array : array
+        Expected to be an unsigned integer array.  0 is background,
+        positive integers label region of interest
+
+    cmap : str or colormap, optional
+        Color map to use, defaults to 'Paired'
+
```

Comment:
Can you add a kwarg for the imshow cmap? Something like `imshow_cmap` would work